public_website/views.py

```diff
@@ -361,9 +361,13 @@ def get(self, request):
             result_id_list = [id['id'] for id in questions.values('id')]
             request.session['result_id_list'] = result_id_list
 
-        paginator = Paginator(questions, 15)
-
+        ITEMS_PER_PAGE = 15
+        paginator = Paginator(questions, ITEMS_PER_PAGE)
         page = request.GET.get('page', 1)
+        # Adding the number of questions/articles being shown based on the page number
+        start_index = (int(page)-1)*ITEMS_PER_PAGE + 1
+        end_index = start_index + ITEMS_PER_PAGE if (start_index + ITEMS_PER_PAGE <= questions.count()) else questions.count()
```

Comment:
its only referencing for questions count thus `end_index` returns to `0` in case when only articles being selected.
Adding a if/else condition for articles count should fix the issue or checking if only articles being in `search_categories`.

Reply:
@abhiprojectz please fix the problem. And remove suggestions.csv from the repo. Thanks.

Fixes #363.